shimmingtoolbox/dicom_to_nifti.py

```diff
@@ -58,6 +58,13 @@ def dicom_to_nifti(path_dicom, path_nifti, subject_id='sub-01', fname_config_dcm
     # Copy original dicom files into nifti_path/sourcedata
     copy_tree(path_dicom, os.path.join(path_nifti, 'sourcedata'))
 
+    # Update the PATH environment variable to include the dcm2niix executable
+    # TODO: Try this out on Windows, path could be Scripts instead of bin
+    if 'ST_DIR' in os.environ and os.path.exists(os.environ['ST_DIR']):
+        os.environ['PATH'] = os.path.join(os.environ['ST_DIR'], 'python', 'bin') + os.pathsep + os.environ['PATH']
+    else:
+        logger.warning("Environment variable ST_DIR not found. Using default path.")
```

Comment:
>  To remedy this, I added dcm2niix to the launchers. As a side effect, this makes Shimming Toolbox's dcm2niix install exposed to the shell, which might be unwanted for some that already have it installed and require specific versions.

Does it expose `dcm2niix` in the shell? From what I understand, changes to `os.environ['PATH']` are only relevant within the Python process. The underlying `PATH` variable doesn't actually get modified, I think!

![image](https://github.com/shimming-toolbox/shimming-toolbox/assets/16181459/4d6d2d2a-b17a-4868-8f8c-06d2f64944f8)

Reply:
This is also my experience. Running `st_dicom_to_nifti` did not affect the PATH in the shell. Thanks!